.gitignore

```diff
@@ -1,2 +1,3 @@
+.vagrantuser
```

Comment:
this the default nugrant config, if you want to overwrite it, you should use system/project wide config.

Reply:
also you can global git ignore vagrant/.vagrantuser, but should always use "vagrant" dir as submodule